integration-cli/docker_cli_run_test.go

```diff
@@ -1743,7 +1743,7 @@ func (s *DockerSuite) TestRunExitOnStdinClose(c *check.C) {
 	name := "testrunexitonstdinclose"
 
 	meow := "/bin/cat"
-	delay := 1
+	delay := 3
```

Comment:
All the exit times I recorded were between 0.9-1.2s, but I can definitely increase this to something higher because it's a failure case and other architectures might be slower than us. I can increase it to 30 just to be consistent with windows